src/DataAccess/DoctrineDonationRepository.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\DataAccess;
+
+use Doctrine\ORM\EntityManager;
+use WMDE\Fundraising\Frontend\Domain\Model\PaymentType;
+use WMDE\Fundraising\Frontend\Domain\Repositories\DonationRepository;
+use WMDE\Fundraising\Entities\Donation as DoctrineDonation;
+use WMDE\Fundraising\Frontend\Domain\Donation;
+
+/**
+ * @license GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class DoctrineDonationRepository implements DonationRepository {
```

Comment:
Dbal is also a library :) Actually I think our other ones should be renamed to Doctine, as they use Doctine ORM, not just DBAL